### PR TITLE
add warpstream example

### PR DESCRIPTION
**Pull request by metaclips**

### CODE REVIEW

File: examples/command/portals/kafka/warpstream/application_team/run_ockam.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -ex
+
+if [[ $OCKAM_DEVELOPER != 'true' ]]; then
+    echo "Set OCKAM_DEVELOPER"
```
Comment (snandam) [inline examples/command/portals/kafka/warpstream/application_team/run_ockam.sh]: why do we need this?

Comment (metaclips) [reply examples/command/portals/kafka/warpstream/application_team/run_ockam.sh]: We actually don't. I was debugging if the environment variable is seen in the docker compose. It's removed now, thanks.

File: examples/command/portals/kafka/warpstream/run.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                -H 'Content-Type: application/json' \
+                -d "$request_body"
+        fi
+    done
+}
+
+# Check to ensure that users have provided their Warpstream API key as an environment variable
+if [[ -z $WARPSTREAM_API_KEY ]]; then
+    echo "ERROR: Please provide your Warpstream API key as an environment variable 'WARPSTREAM_API_KEY'" && exit 1;
+fi
```
Comment (mrinalwadhwa) [inline examples/command/portals/kafka/warpstream/run.sh]: Let's make this an argument that is provided as on the command line in place of an environment variable.

**Review by snandam — DISMISSED** [review]

**Review by etorreborre — DISMISSED** [review]

File: examples/command/portals/kafka/warpstream/run.sh (hunk 2 above)
Comment (metaclips) [reply examples/command/portals/kafka/warpstream/run.sh]: Done

**Review by etorreborre — APPROVED** [review]

**Review by SanjoDeundiak — APPROVED** [review]

**Pull request merged**